.env.development

```diff
@@ -1 +1 @@
-REACT_APP_CLUSTER=devnet
+REACT_APP_CLUSTER=mainnet-beta
```

Comment:
this is maybe not desirable?

Reply:
you can't do any harm but I'll change it